app/controllers/appointments.js

```diff
@@ -130,10 +130,20 @@ export default GoodcityController.extend({
 
     showSpecial() {
       this.set("selectedTab", "special");
+      _.map(this.get("specialSlotsByDate"), days => {
+        if (days.get("showOptionsMenu")) {
+          days.set("showOptionsMenu", !days.get("showOptionsMenu"));
+        }
+      });
     },
 
     showDefault() {
       this.set("selectedTab", "presets");
+      _.map(this.get("presetsByWeekDay"), days => {
+        if (days.get("showOptionsMenu")) {
+          days.set("showOptionsMenu", !days.get("showOptionsMenu"));
+        }
+      });
```

Comment:
`map` is used to transform an array by returning a new element for each item, if you don't return anything inside the predicate, then you should use `each` or `forEach` instead

```suggestion
      _.each(this.get("presetsByWeekDay"), days => days.set("showOptionsMenu", false))
```

or

```suggestion
      this.get("presetsByWeekDay").forEach(days => days.set("showOptionsMenu", false))
```